proto/errorpb.proto

```diff
@@ -19,9 +19,13 @@ message KeyNotInRegion {
     optional bytes end_key     = 4;
 }
 
+message EpochOutdated {
+}
+
 message Error {
     optional string message                         = 1;
     optional NotLeaderError not_leader              = 2;
     optional RegionNotFoundError region_not_found   = 3;
     optional KeyNotInRegion key_not_in_region       = 4;
+    optional EpochOutdated epoch_outdated           = 5;
```

Comment:
When user tell if an error is `EpochOutdated`, he will write `has_epoch_outdated`, which just sound like a normal question: "Has epoch (be) outdated?"